SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -2539,16 +2540,21 @@ UpdateDeletePage (
   EFI_SIGNATURE_LIST  *CertList;
   EFI_SIGNATURE_DATA  *Cert;
   UINT32              ItemDataSize;
-  CHAR16              *GuidStr;
-  EFI_STRING_ID       GuidID;
+  CHAR16              *CertificateInfoStr;
+  CHAR8               *CertificateInfoStr8;
+  EFI_STRING_ID       CertificateInfoID;
   EFI_STRING_ID       Help;
+  UINTN CertificateInfoStrSize = 100;

```

Comment:
What about the dynamic allocation? Krystian already explained here: https://github.com/Dasharo/edk2/pull/149#discussion_r1848294097